relentless/core.py

```diff
@@ -15,11 +15,20 @@ class Interpolator:
 
     Parameters
     ----------
-    x : array_like
+    x : float or array_like
```

Comment:
Sorry, this is my fault. Clearly the interpolation fails if x is a scalar, so revert this to `array_like`.